files/docker-compose.yml

```
@@ -0,0 +1,104 @@
+---
+version: "3.7"
+
+secrets:
+  admiral_yml:
+    file: ./secrets/admiral.yml
+  mongo_root_passwd_txt:
+    file: ./secrets/mongo-root-passwd.txt
+  redis_conf:
+    file: ./secrets/redis.conf
+  sslmate_api_key_txt:
+    file: ./secrets/sslmate-api-key.txt
+
+x-admiral-template: &admiral-template
+  build:
+    context: .
+    dockerfile: Dockerfile-admiral
+  deploy:
+    mode: replicated
+    replicas: 5
+  environment:
+    ADMIRAL_CONFIG_FILE: "/run/secrets/admiral.yml"
+    ADMIRAL_CONFIG_SECTION: dev-mode
+    ADMIRAL_WORKER_NAME: dev
+  image: admiral
+  init: true
+  secrets:
+    - source: admiral_yml
+      target: admiral.yml
+    - source: sslmate_api_key_txt
+      target: sslmate-api-key.txt
+  volumes:  # map for development only, comment out otherwise
+    - ./src/admiral:/usr/src/admiral/admiral
+
+services:
+  celery-flower:
+    environment:
+      # TODO: https://github.com/cisagov/admiral/issues/19
+      # move into the secrets raft
+      CELERY_BROKER_URL: "redis://:fruitcake@redis:6379/0"
+    # TODO: https://github.com/cisagov/admiral/issues/18
+    # replace this outdated image
+    image: crgwbr/docker-celery-flower
+    ports:
+      - "5555:5555"
+
+  cert-worker:
+    <<: *admiral-template
+    environment:
+      ADMIRAL_CONFIG_SECTION: cert-worker
+      ADMIRAL_WORKER_NAME: cert
+
+  mongo:
+    environment:
+      MONGO_INITDB_DATABASE: certs
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD_FILE: /run/secrets/mongo_root_passwd_txt
+    image: mongo:4.2
+    restart: always
+    secrets:
+      - source: mongo_root_passwd_txt
+    volumes:
+      - ./init/mongo-init.js:/docker-entrypoint-initdb.d/mongo-init.js:ro
+
+  mongo-express:
+    environment:
+      ME_CONFIG_MONGODB_SERVER: mongo
+      ME_CONFIG_MONGODB_ADMINUSERNAME: root
+      ME_CONFIG_MONGODB_ADMINPASSWORD: example
+    image: mongo-express:0.54
+    ports:
+      - "8083:8081"
+    restart: always
+
+  redis:
+    command:
+      - "redis-server"
+      - "/run/secrets/redis_conf"
+    image: redis:7.0-alpine
+    ports:
+      - host_ip: 127.0.0.1
+        mode: host
+        protocol: tcp
+        published: 6379
+        target: 6379
+    secrets:
+      - source: redis_conf
+
+  redis-commander:
+    depends_on:
+      - redis
+    environment:
+      REDIS_HOSTS: "default:redis:6379:0:fruitcake"
+    hostname: redis-commander
+    image: ghcr.io/joeferner/redis-commander:0.8
```

I see that [`joeferner/redis-commander:0.8.1`](https://github.com/joeferner/redis-commander/pkgs/container/redis-commander/53727781?tag=0.8.1) has been out for 10 months, so we might as well use that, though you should test with it to ensure nothing breaks:
```suggestion
    image: ghcr.io/joeferner/redis-commander:0.8.1
```